### DISCUSSION TITLE
Agent Nick Michaels (CARTEL) [U]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#80] [Card Balance]
Digital card :
UNIQUE, PROFANE
Will never become a HERETIC. Gains the affiliation of the deck he is played in.
F: 5 S: 5 A: 5 V: 5

My suggestion :
UNIQUE, PROFANE
Will never become a HERETIC. Gains the affiliation of the deck he is played in.
When played, search through your Draw pile and attach to him for no Action cost any one equipment  with his affiliation.
F: 4 S: 4 A: 4 V: 4

**Comment by Sol-au-fein** [comment]
Agent Nick Michaels (CARTEL)
F: 5 S: 5 A: 5 V: 5
UNIQUE PROFANE
Gains affiliation of the deck he is played in.
While is Agent Nick Michaels in play, each warrior in opponent's squad suffers -1/-1/-1 penalty.

As allways im taking inspiration in the lore. So my idea is based on his freelancer and infiltration background. Because he knows all strenghts and weaknesses of every corporation, all opponent's warriors suffers -1/-1/-1  debuff. I think it is not too powerfull and reflects lore very well. 

`Nick Michaels is a name known throughout the corporations as both friend and foe. A lost bid for this enigmatic freelancer means panic in the security division; for no one knows who he is working for until it is too late. His powers of infiltration and corporate knowledge makes him on of the highest paid freelancer in the bussiness. His value is such that each corporation attempts to win his favor by granting him honorable membership - meaning he is a member of all five megacorporations.
Special Rules: Nick Michaels is known for being extremely lucky. Treat him as if he has Lucky Fate special ability.(50% of recieved hits misses) `

**Comment by Michalx11** [comment]
I like this idea

**Comment by TomekPL-1** [comment]
Agent Nick Michaels (CARTEL)
F: 5 S: 5 A: 5 V: 5
UNIQUE PROFANE
Gains affiliation of the deck he is played in. When he become wounded during combat flip a coin - Heads wound is ignored and opponent gain 1PP, Tails Nick is wounded as normal and attacking warrior gain permanently +1V.

From the lore Sol-au-fein quoted : 
"Treat him as if he has Lucky Fate special ability.(50% of recieved hits misses) " - and he is so famous that if someone managed to wound his status autoamtically becomes higher +1V
